data/library/tlg0059/tlg003/metadata.json

```diff
@@ -11,24 +11,23 @@
   ],
   "versions": [
     {
-      "urn": "urn:cts:greekLit:tlg0059.tlg003.jkt:",
+      "urn": "urn:cts:greekLit:tlg0059.tlg003.jkt-1:",
       "node_kind": "version",
       "version_kind": "edition",
-      "first_passage_urn": "urn:cts:greekLit:tlg0059.tlg003.jkt:001-009",
+      "first_passage_urn": "urn:cts:greekLit:tlg0059.tlg003.jkt-1:001-009",
       "citation_scheme": ["speech", "sentence"],
       "label": [
         {
           "lang": "eng",
-          "value": "Crito"
+          "value": "Crito (jtauber Edition)"
```

Comment:
Definitely deserves a `James Tauber Edition` here.

Reply:
I was going for the location in GitHub rather than making a personal editorial claim :-)

Unlike the Open Apostolic Fathers where I actually edited the text, this is still the Burnet text, it's just chunked completely differently (by me).